src/main/scala/com/twitter/finagle/postgres/codec/PgCodec.scala

```diff
@@ -186,7 +164,10 @@ class PgClientChannelHandler(
   sslEngineFactory: SslClientEngineFactory,
   sslConfig: Option[SslClientConfiguration],
   val useSsl: Boolean
-) extends SimpleChannelHandler {
+) extends MessageToMessageCodec[BackendMessage, Object] {
+
+  private val allocator = new PooledByteBufAllocator()
```

Comment:
Why not using Finagle's default allocator? It's in [this stack param](https://github.com/twitter/finagle/blob/develop/finagle-netty4/src/main/scala/com/twitter/finagle/netty4/param/Allocator.scala).

Reply:
Thanks, I'll see if I can figure that out!

Reply:
Done!